### DISCUSSION TITLE
Projects Beta: Allow a project to be public

### DISCUSSION THREAD

**Discussion by ashleykolodziej** [community/community#5681] [Projects and Issues] [answered]
Hey there!

I have issues on private repos added to my project beta board. I would like the board to be able to be made public so that others on my team can see the status of the work and the amount of work we have. Ideally, this board would show the titles of issues from both private and public repos. At the issue level, if you clicked on a private issue, you would not see the details.

I really want finer grained issue control as mentioned in https://github.com/github/feedback/discussions/5415, but my biggest need is definitely for visibility of issue status outside my developer and project management teams, who have GitHub accounts. The Projects beta would be a HUGE help with this if I could just set them to public!

**Comment by Harmon758** [comment, marked as answer]
You can change the visibility of projects to be public now 🎉 

https://github.blog/2021-10-27-everything-new-from-universe-2021/
